cves/CVE-2012-2779.yml

```diff
@@ -97,7 +101,8 @@ fixes:
   note: ''
```

Comment:
Did you look at the fix commits? What did the authors do to fix the issue? What changed in the code?